addons/parachute/functions/fnc_showAltimeter.sqf

```diff
@@ -17,32 +17,33 @@
 
 params ["_unit"];
 
-(["ACE_Altimeter"] call BIS_fnc_rscLayer) cutRsc ["ACE_Altimeter", "PLAIN", 0, true];
+"ACE_Altimeter" cutRsc ["ACE_Altimeter", "PLAIN", 0, true];
 if (isNull (uiNamespace getVariable ["ACE_Altimeter", displayNull])) exitWith {};
 
 GVAR(AltimeterActive) = true;
 
 [{
-    if (!GVAR(AltimeterActive)) exitWith {[_this select 1] call CALLSTACK(CBA_fnc_removePerFrameEventHandler)};
+    if (!GVAR(AltimeterActive)) exitWith {[_this select 1] call CBA_fnc_removePerFrameEventHandler};
     disableSerialization;
     (_this select 0) params ["_display", "_unit", "_oldHeight", "_prevTime"];
-    if !("ACE_Altimeter" in assignedItems _unit) exitWith {[_this select 1] call CALLSTACK(CBA_fnc_removePerFrameEventHandler); call FUNC(hideAltimeter)};
+    if !("ACE_Altimeter" in assignedItems _unit) exitWith {[_this select 1] call CBA_fnc_removePerFrameEventHandler; call FUNC(hideAltimeter)};
 
     private _HeightText = _display displayCtrl 1100;
     private _DecendRate = _display displayCtrl 1000;
     private _TimeText = _display displayCtrl 1001;
+
     private _hour = floor daytime;
     private _minute = floor ((daytime - _hour) * 60);
 
     private _height = ((getPosASL _unit) select 2) + EGVAR(common,mapAltitude);
     private _curTime = CBA_missionTime;
     private _timeDiff = _curTime - _prevTime;
-    private _descentRate = if(_timeDiff > 0) then {floor((_oldHeight - _height) / _timeDiff)} else {0};
+    private _descentRate = if (_timeDiff > 0) then {floor((_oldHeight - _height) / _timeDiff)} else {0};
 
-    _TimeText ctrlSetText (format ["%1:%2",[_hour, 2] call EFUNC(common,numberToDigitsString),[_minute, 2] call EFUNC(common,numberToDigitsString)]);
```

Comment:
We want the old way, it pads zeros to the left
[5, 2] call ace_common_fnc_numberToDigitsString = "05"
5 toFixed 2 ="5.00"